tests/unit/importer/test_importer.py

```diff
@@ -102,8 +109,10 @@ def test_import_file__link_supplementary_files_to_project(self, mock_link_entity
         submission, _ = self.importer.import_file(file_path='path', submission_url='url')
 
         # then:
-        self.assertEqual([args.kwargs.get('relationship') for args in mock_link_entity.call_args_list],
-                         ['submissionEnvelopes', 'supplementaryFiles', 'project'])
+        # assertCountEqual also asserts elements are the same,
+        # see doc https://docs.python.org/3.2/library/unittest.html#unittest.TestCase.assertCountEqual
+        self.assertCountEqual([args.kwargs.get('relationship') for args in mock_link_entity.call_args_list],
```

Comment:
What is this testing? I am afraid it is not very clear.

Reply:
it's asserting that the link_entity mock should have been called for these relationships, do you have any suggestions to make it clearer?

Reply:
I've changed the assertions to be:
```
# then:
        self.assertIn('submissionEnvelopes', relationships,
                      'The submission envelope should be linked as project\'s submission envelopes.')
        self.assertIn('supplementaryFiles', relationships,
                      'The files should be the linked to project\'s supplementary files.')
        self.assertIn('project', relationships,
                      'The supplementary file should have a linked to the project')
```
I'm not sure how else to capture and check that those links happened. Hopefully, that's a bit clearer. Please note, I moved that test function is now in the new test suite file.

Reply:
Those 3 are good, because they explain to the reader the domain rules you are checking.

You could also have an assertion to check the existence of a relationship and call it 3 times. In other words, less low level unitests api, and more domain language in the tests.